src/main/kotlin/com/sourcegraph/cody/context/RemoteRepoInsight.kt

```diff
@@ -369,6 +380,8 @@ class RemoteRepoCompletionContributor : CompletionContributor(), DumbAware {
             prefixedResult.restartCompletionOnAnyPrefixChange()
             try {
               runBlockingCancellable {
+                // TODO: Extend repo search to consult Cody Ignore and denote repositories that are
+                // ignored.
```

Comment:
☝️

Reply:
I looked at VSCode and it isn't doing any badging here (yet... @valerybugakov might be looking at that) so I think this is fine to leave as a TODO for now. If you add ignored repos, you get feedback about it in the tree view.